Fix build breakage on CentOS 6

As of e551fbcfc2dd97c725783504f5e4e3da32bba512 (possibly as of a0c6f0ec) the `master` branch is broken for attempted builds on CentOS 6.

```
make[2]: Entering directory `/home/bmah/ESnet/src/iperf/src'
  CC       iperf3-main.o
In file included from main.c:46:
iperf_api.h:48: error: redefinition of typedef ‘iperf_size_t’
iperf.h:71: note: previous declaration of ‘iperf_size_t’ was here
```

We need to fix this before release.

--- a/src/iperf.h
+++ b/src/iperf.h
@@ -68,7 +68,9 @@
 #include <openssl/evp.h>
 #endif // HAVE_SSL
 
+#if !defined(__IPERF_API_H)
 typedef uint64_t iperf_size_t;
+#endif // __IPERF_API_H
 
 struct iperf_interval_results
 {

--- a/src/iperf_api.h
+++ b/src/iperf_api.h
@@ -45,7 +45,9 @@ struct iperf_interval_results;
 struct iperf_stream;
 struct iperf_time;
 
+#if !defined(__IPERF_H)
 typedef uint64_t iperf_size_t;
+#endif // __IPERF_H
 
 /* default settings */
 #define Ptcp SOCK_STREAM

